### PR TITLE
Check code quality for UI

### LINKED ISSUE #328: [UI Package] Check code quality
Check util, widget, and rest of files in model package

### PR DESCRIPTION
**Pull request by tankangliang**
## Description

Fixes #328 

## Testing

Automated testing

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -205,7 +204,7 @@ public void updateTagNoteMapWithNote(Set<Tag> newTags, Note note) {
     }
 
     /**
-     * Initialises TagNoteMap from Clients notes and Country notes.
+     * Initialises TagNoteMap from client notes and country notes.
```
Comment (LeeEnHao) [inline src/main/java/seedu/address/model/ModelManager.java]:
```suggestion
     * Initialises {@code TagNoteMap} from client notes and country notes.
```

File: src/main/java/seedu/address/model/widget/WidgetModel.java [hunk 2]
```diff
@@ -3,19 +3,19 @@
...
+ * This is the API for interacting with the View Box widget in the TBM application. The view box widget model
  * is encapsulated within the TBM model itself.
  */
 public interface WidgetModel {
 
     /**
      * Static factory.
      *
-     * @return WidgetModel implemented by WidgetModelMoanager.
+     * @return WidgetModel implemented by WidgetModelManager.
```
Comment (LeeEnHao) [inline src/main/java/seedu/address/model/widget/WidgetModel.java]:
```suggestion
     * @return WidgetModel implemented by {@code WidgetModelManager}.
```

**Review by LeeEnHao — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/logic/commands/CommandResult.java [hunk 3]
```diff
@@ -88,14 +88,14 @@ public String getWidgetViewOptionAsString() {
...
+        return widgetViewOption.isClientView();
     }
 
     /**
      * Returns true if the UI should display the country notes view.
      */
     public boolean shouldDisplayCountryNote() {
-        return widgetViewOption.isCountryNote();
+        return widgetViewOption.isCountryNoteView();
     }
```
Comment (qwoprocks) [inline src/main/java/seedu/address/logic/commands/CommandResult.java]: Add `view` to the func names?

File: src/main/java/seedu/address/model/Model.java [hunk 4]
```diff
@@ -129,19 +129,19 @@
...
 
     /**
-     * Edits a Client Note associated to a particular Client.
+     * Edits a note associated to a particular client.
      *
-     * @param associatedClient Client associated to the note to be edited.
+     * @param associatedClient Client that the note belongs to.
      * @param noteToEdit     Note to be edited.
-     * @param  newNote     newNote to be used.
+     * @param  newNote     New note to be used.
```
Comment (qwoprocks) [inline src/main/java/seedu/address/model/Model.java]: I think the indent should be matched between params. Either use one space between segments or tab to align among params

File: src/main/java/seedu/address/model/TbmManager.java [hunk 5]
```diff
@@ -110,20 +108,20 @@ public void removeClient(Client key) {
     }
 
     /**
-     * Checks whether the given country has the given countryNote.
+     * Checks whether the given country has the given note {@code countryNote}.
      *
-     * @param countryNote The given countryNote
-     * @return True if the given country has the given countryNote.
+     * @param countryNote The given country note.
+     * @return True if the given country has the given note {@code countryNote}.
```
Comment (qwoprocks) [inline src/main/java/seedu/address/model/TbmManager.java]:
```suggestion
     * @return True if the given country has the given {@code countryNote}.
```

File: src/main/java/seedu/address/model/TbmManager.java [hunk 6]
```diff
@@ -110,20 +108,20 @@ public void removeClient(Client key) {
...
      */
     public boolean hasCountryNote(CountryNote countryNote) {
         requireNonNull(countryNote);
         return countryNotesManager.hasCountryNote(countryNote);
     }
 
     /**
-     * Adds the given countryNote to the given country.
+     * Adds the given note {@code countryNote} into the notes manager for country.
```
Comment (qwoprocks) [inline src/main/java/seedu/address/model/TbmManager.java]:
```suggestion
     * Adds the given {@code countryNote} into the notes manager for country.
```

File: src/main/java/seedu/address/model/widget/WidgetModel.java (hunk 2 above)
Comment (qwoprocks) [inline src/main/java/seedu/address/model/widget/WidgetModel.java]: lmao nice catch

File: src/main/java/seedu/address/model/widget/WidgetModel.java [hunk 7]
```diff
@@ -3,19 +3,19 @@
 import seedu.address.model.client.Client;
 
 /**
- * This is the api for interacting with the proposed View Box widget in the TBM application. The view box widget model
+ * This is the API for interacting with the View Box widget in the TBM application. The view box widget model
  * is encapsulated within the TBM model itself.
```
Comment (qwoprocks) [inline src/main/java/seedu/address/model/widget/WidgetModel.java]: Maybe change "TBM model" to something else

File: src/test/java/guitests/guihandles/StatusBarFooterHandle.java [hunk 8]
```diff
@@ -18,7 +18,7 @@
     /**
      * Constructor for handler.
      *
-     * @param statusBarFooterNode
+     * @param statusBarFooterNode node to be passed in.
```
Comment (qwoprocks) [inline src/test/java/guitests/guihandles/StatusBarFooterHandle.java]:
```suggestion
     * @param statusBarFooterNode Node to be passed in.
```

File: src/main/java/seedu/address/ui/CountryNoteListPanel.java [hunk 9]
```diff
@@ -132,4 +133,13 @@ public void addNoteListCard(NoteListCard noteListCard) {
         }
     }
 
+    /**
+     * Appends " notes" to the end of country to be used as header string.
+     * @param country Country to be formatted as string.
```
Comment (qwoprocks) [inline src/main/java/seedu/address/ui/CountryNoteListPanel.java:142]: Needs a new line between desc and param

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 10]
```diff
@@ -79,7 +79,7 @@ private void setAccelerators() {
 
     /**
      * Sets the accelerator of a MenuItem.
-     * @param keyCombination the KeyCombination value of the accelerator
+     * @param keyCombination the KeyCombination value of the accelerator.
```
Comment (qwoprocks) [inline src/main/java/seedu/address/ui/MainWindow.java:86]: Needs a space between desc and param

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 11]
```diff
@@ -93,25 +98,25 @@
     }
 
     public static List<CountryNote> getSampleCountryNotes() {
```
Comment (qwoprocks) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:106]: need javadoc here, or just make it private? Probably same for function above

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 12]
```diff
@@ -93,25 +98,25 @@
...
-        third.setTags(thirdTags);
+        Set<Tag> thirdCountryNoteTags = new HashSet<>();
+        thirdCountryNoteTags.add(new Tag("intercultural"));
+        thirdCountryNote.setTags(thirdCountryNoteTags);
 
-        return Arrays.asList(first, second, third);
+        return Arrays.asList(firstCountryNote, secondCountryNote, thirdCountryNote);
     }
 
     public static ReadOnlyTbmManager getSampleTbmManager() {
```
Comment (qwoprocks) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:131]: need javadoc here

File: src/main/java/seedu/address/model/ModelManager.java [hunk 13]
```diff
@@ -60,10 +60,9 @@ public ModelManager(ReadOnlyTbmManager tbmManager, ReadOnlyUserPrefs userPrefs)
...
         sortedFilteredCountryNotes = new SortedList<>(filteredCountryNotes,
-                Comparator.comparing(CountryNote::getCountry)
-                        .thenComparingInt(filteredCountryNotes::indexOf));
+                Comparator.comparing(CountryNote::getCountry).thenComparingInt(filteredCountryNotes::indexOf));
         this.tagNoteMap = new TagNoteMap();
-        this.initialiseTagNoteMap(); // init TagNoteMap upon construction of modelManager
+        this.initialiseTagNoteMap(); // init TagNoteMap upon construction of ModelManager
     }
 
     public ModelManager() {
```
Comment (qwoprocks) [inline src/main/java/seedu/address/model/ModelManager.java:71]: Need javadoc

**Review by qwoprocks — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/model/ModelManager.java (hunk 1 above)
Comment (tankangliang) [reply src/main/java/seedu/address/model/ModelManager.java]: will update

File: src/main/java/seedu/address/model/widget/WidgetModel.java (hunk 2 above)
Comment (tankangliang) [reply src/main/java/seedu/address/model/widget/WidgetModel.java]: will update

File: src/main/java/seedu/address/model/ModelManager.java (hunk 13 above)
Comment (tankangliang) [reply src/main/java/seedu/address/model/ModelManager.java:71]: done

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 12 above)
Comment (tankangliang) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:131]: done

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 11 above)
Comment (tankangliang) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:106]: done

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 10 above)
Comment (tankangliang) [reply src/main/java/seedu/address/ui/MainWindow.java:86]: done

File: src/main/java/seedu/address/ui/CountryNoteListPanel.java (hunk 9 above)
Comment (tankangliang) [reply src/main/java/seedu/address/ui/CountryNoteListPanel.java:142]: done

File: src/test/java/guitests/guihandles/StatusBarFooterHandle.java (hunk 8 above)
Comment (tankangliang) [reply src/test/java/guitests/guihandles/StatusBarFooterHandle.java]: done

File: src/main/java/seedu/address/model/widget/WidgetModel.java (hunk 7 above)
Comment (tankangliang) [reply src/main/java/seedu/address/model/widget/WidgetModel.java]: removed lmao

File: src/main/java/seedu/address/model/TbmManager.java (hunk 6 above)
Comment (tankangliang) [reply src/main/java/seedu/address/model/TbmManager.java]: done

File: src/main/java/seedu/address/model/TbmManager.java (hunk 5 above)
Comment (tankangliang) [reply src/main/java/seedu/address/model/TbmManager.java]: done

File: src/main/java/seedu/address/model/Model.java (hunk 4 above)
Comment (tankangliang) [reply src/main/java/seedu/address/model/Model.java]: done

File: src/main/java/seedu/address/logic/commands/CommandResult.java (hunk 3 above)
Comment (tankangliang) [reply src/main/java/seedu/address/logic/commands/CommandResult.java]: done

File: src/main/java/seedu/address/model/TbmManager.java [hunk 14]
```diff
@@ -32,18 +32,19 @@
...
     }
 
+    /**
+     * Creates an empty TbmManager without any existing data.
+     */
     public TbmManager() {}
 
     /**
-     * Creates an TbmManager using the Clients in the {@code toBeCopied}
+     * Creates an TbmManager using the clients in the {@code toBeCopied}
```
Comment (qwoprocks) [inline src/main/java/seedu/address/model/TbmManager.java]:
```suggestion
     * Creates an TbmManager using the data in the {@code toBeCopied}
```

**Review by qwoprocks — APPROVED** [review]
LGTM except for one comment

**Review by LeeEnHao — APPROVED** [review]
Looks good

**Pull request merged**